root/work/edit_form.tt

```diff
@@ -100,13 +100,13 @@
     [%- iswc_bubble(link_entity(work)) -%]
 
     <div class="bubble" id="type-bubble">
-      <span id="type-bubble-default">
+      <span id="type-bubble-default" [% IF form.field('type_id').value %] style="display:none" [% END %]>
         [%- l('Select any type from the list to see its description. If the work
                doesn’t seem to match any type, just leave this blank.') -%]
       </span>
       [% FOREACH id IN form.field('type_id').options %]
          [%- id = id.value -%]
-         <span style="display:none" class="type-bubble-description" id="type-bubble-description-[% id %]">
+         <span [% IF form.field('type_id').value != id %] style="display:none" [% END %] class="type-bubble-description" id="type-bubble-description-[% id %]">
            <strong>[%- add_colon(l('Description')) -%] </strong>[%- work_types.$id.l_description -%]
```

Comment:
I would not mind removing _Description:_ by the way as it is obvious from work types descriptions: “An aria is…”

Reply:
I made the same comment to #1741 which seems to be more appropriate to address that. Let's close this conversation.